src/controllers/monster.js

```diff
@@ -425,6 +425,16 @@ class Monster extends Controller {
 				return []
 			}
 
+			if (data.display_pokemon_id) {
```

Comment:
trolly webhooks might send pokemon_id as display_pokemon_id, a simple `!=` would make sure it never happens

Reply:
agreed, added